examples/remote_procedure/toxicity_detection/webapp/src/app.jsx

```diff
@@ -62,6 +63,24 @@ function RemoteProcedureApp() {
             handleSubmit={handleSubmit}
             handleToxicityCalculation={handleToxicityCalculation}
           />
+          <Tips
+            handleSubmit={(tipData) => createTip(tipData.header, tipData.text)}
```

Comment:
isn't this essentially a no-op? createTip doesn't have any side effects right? should we `console.log` instead?

Reply:
where is `createTip` coming from? missing import?

Reply:
Yeah, it should be console.log